lib/net.js

```diff
@@ -370,13 +370,13 @@ Socket.prototype._final = function(cb) {
   const err = this._handle.shutdown(req);
 
   if (err === 1)  // synchronous finish
-    return afterShutdown.call(req, 0);
+    return afterShutdown.call(req);
   else if (err !== 0)
     return this.destroy(errnoException(err, 'shutdown'));
 };
 
 
-function afterShutdown(status) {
+function afterShutdown() {
```

Comment:
While this is currently ignored, it does receive an argument when called from C++, and if you remove the parameter I’d prefer having at least a comment (e.g. `/* status */`) that clarifies that that parameter exists.